src/interface/TWTezosSigner.cpp

```diff
@@ -51,7 +53,9 @@ TW_Tezos_Proto_SigningOutput TWTezosSignerSign(TW_Tezos_Proto_SigningInput data)
     auto signedBytesHex = signer.signOperationList(key, operationList);
```

Comment:
Can we generate this as a byte array to start with?

Reply:
Done.